sdk/python/kfp/v2/proto/pipeline_spec_pb2.py

```diff
@@ -45,28 +45,23 @@
   full_name='ml_pipelines.PrimitiveType.PrimitiveTypeEnum',
   filename=None,
   file=DESCRIPTOR,
-  create_key=_descriptor._internal_create_key,
```

Comment:
JFYI: Removing `_descriptor._internal_create_key` **fixes** importing this file on my system when using latest protobuf package.

Reply:
I want to minimize manual modification whenever possible. Do you mind giving the protoc version here so that I can try reproduce it? Thanks